src/doctor/rules/hypertable.py

```diff
@@ -49,13 +47,15 @@
 """
 
 @doctor.register
-@dataclass
 class HypertableCandidate(doctor.Rule):
     """Detect candidate hypertable."""
 
     query: str = CANDIDATE_QUERY
     detail: str = CANDIDATE_DETAIL
     message: str = "Table {table} might benefit from being transformed to a hypertable."
+    dependencies: dict = {
```

Comment:
It seems backwards to require timescaledb to be installed to say whether a table should be a hypertable or not. I assume that we would want to run `timescale-doctor` against a plain postgres database, and get a statement about which tables should be converted to hypertables.

Reply:
Agree. Will investigate.